docs/design/admin_control_over_spec.md

```diff
@@ -0,0 +1,69 @@
+Admin users may want to further restrict non admin users operations, like forcing a specific time to live (TTL) for NonAdminBackups associated Velero Backups.
+This design enables admin users to set custom default values for NonAdminBackup/NonAdminRestore spec fields, which can not be overridden.
+
+## Goals
+
+Enable admin users to
+- set custom default values for NonAdminBackup spec.backupSpec fields, which can not be overridden
+- TODO restore
+
+## Non Goals
+
+- Show NonAdminBackup spec.backupSpec fields/TODO NonAdminRestore custom default values to non admin users
+- Do not allow non admin users to create NonAdminBackup/NonAdminRestore with overridden defaults (but they will not validated by NAC, with an error explaining why in the object statuses and in NAC logs)
+- Allow admin users to set second level defaults (for example, NonAdminBackup `spec.backupSpec.labelSelector` can have a custom default value, but not just `spec.backupSpec.labelSelector.matchLabels`)
+
+
+## High-Level Design
+
+A field will be added to OADP DPA object. With it, admin users will be able to select which NonAdminBackup `spec.backupSpec` fields have custom default (and enforced) values. NAC will respect the set values. If a NonAdminBackup is created with fields overriding any enforced values, it will fail validation prior to creating an associated Velero Backup. If admin user changes any enforced field value, NAC Pod is recreated to always be up to date with admin user enforcements.
+
+TODO restore
+
+
+## Detailed Design
+
+Field `spec.nonAdmin.enforceBackupSpec`, of the same type as the Velero Backup Spec, will be added to OADP DPA object.
+
+With it, admin users will be able to select which NonAdminBackup `spec.backupSpec` fields have custom default (and enforced) values.
+
+To avoid mistakes, not all fields will be able to be enforced, like `IncludedNamespaces`, that could break NAC usage.
+
+NAC will respect the set values by reading DPA field during startup.If admin user changes any enforced field value, NAC Pod is recreated (and only NAC Pod) to always be up to date with admin user enforcements.
+
+If a NonAdminBackup is created with fields overriding any enforced values, it will fail validation prior to creating an associated Velero Backup.
+
+If NonAdminBackup respects enforcement, the created associated Velero Backup will have the enforced spec field values.
+
+Enforcement is done dynamically. If new field is added to Velero Backup Spec, it will be presented to user without code changes. If a field changes type/or default value, tests will warn us.
+
+## Alternatives Considered
+
+Instead of using a DPA field, using a ConfigMap was considered. Since users would not have type assertion when creating those ConfigMap and parsing it would be harder in NAC side, it was discarded.
+
+## Security Considerations
+
+No security considerations.
+
+## Compatibility
+
+No compatibility issues.
+
+## Implementation
+
```

Comment:
Lets add some text here explaining the logic that's getting implemented. For instance you could say:
- Add Enforced Backup Spec to NAB reconciler struct
- Modify ValidateSpec function to use EnforcedBackup spec and apply that to users' NAB request